### DISCUSSION TITLE
Can it be assumed modules with no "interval" configuration are events-based?

### DISCUSSION THREAD

**Discussion by rieje** [Alexays/Waybar#3109] [Q&A]
Can it be assumed modules with no "interval" configuration are events-based and therefore efficient to use? I would like to add a bunch of modules and some are not essential but I also don't want them permanently polling and running external commands especially if the output is usually unchanged.

**Comment by zhjihuang** [comment]
I'd like to know this as well. I'm trying to create a custom module like the sway scratchpad module but for zellij sessions. The scratchpad updates instantly while I'm currently resorting to polling intervals which eats up battery life and has a delay.
